test/getDBOptions.spec.js

```diff
@@ -8,7 +8,7 @@ const {
   ConnectionFactory,
 } = require('./setup-database');
 
-describe('db.getOptions()', () => {
+describe('options.get()', () => {
```

Comment:
We should probably add a test for getConfig

Reply:
done